requirements-test.txt

```diff
@@ -1,4 +1,10 @@
-httpretty
-nose
-requests>=1.0,<2.0
-pdftables>=0.0.4
+httpretty==0.8.6
+nose==1.3.6
+requests==2.2.1
+# pdftables==0.0.4
```

Comment:
We're unlikely to update pdftables to python3 any time soon; it seemed an awful lot of work for minimal gain.